Add [SPC]bb map to list all buffers in fzf layer

jetm opened the pull request:
fzf layer works better than denite and unite layers. This commit add
`[SPC]bb` map in fzf layer.

denite layer used to enable `[SPC]bb` to navigate in all buffers.
This change add same functionality but in fzf layer.

I didn't write tests for this change. Don't know how to do it. I would have created this map in my init.vim file, but an issue in how to use [SPC].  

# PR Prelude

Thank you for working on SpaceVim! :)

**Please complete these steps and check these boxes (by putting an `x` inside
the brackets) _before_ filing your PR:**

- [X] I have read and understood SpaceVim's [CONTRIBUTING][cont] document.
- [X] I have read and understood SpaceVim's [CODE_OF_CONDUCT][code] document.
- [ ] I have included tests for the changes in my PR. If not, I have included a
  rationale for why I haven't.
- [X] **I understand my PR may be closed if it becomes obvious I didn't
  actually perform all of these steps.**

# Why this change is necessary and useful

[Please explain **in detail** why the changes in this PR are needed.]

[cont]: https://github.com/SpaceVim/SpaceVim/blob/master/CONTRIBUTING.md
[code]: https://github.com/SpaceVim/SpaceVim/blob/master/CODE_OF_CONDUCT.md

wsdjeg commented:
@jetm Thanks for your PR, but in fzf layer most key bindings are wrapped via  vim script instead of using plugin fzf.vim, I recommand to write a simple function to implement this feature.

jetm commented:
@wsdjeg oh, I see. Why no to reuse fzf.vim? Do you have an example how it was done in SP?

wsdjeg commented:
@jetm you can only add this command to the end of fzf layer in SpaceVim

```vim
command! Fzfbuffers call <SID>buffers()
function! s:open_buffer(e) abort
  execute 'buffer' matchstr(a:e, '^[ 0-9]*')
endfunction
function! s:buffers() abort
  let s:source = 'buffers'
  function! s:buffer_list() abort
    return split(s:CMP.execute('buffers'), '\n')
  endfunction
  call fzf#run({
        \   'source':  reverse(<sid>buffer_list()),
        \   'sink':    function('s:open_buffer'),
        \   'options': '+m',
        \   'down': '40%'
        \ })
endfunction
```
then you can use:

```
call SpaceVim#mapping#space#def('nnoremap', ['b', 'b'], 'Fzfbuffers', 'buffer list', 1)
```

wsdjeg commented:
ping @jetm hello, what is the status of this PR? Should I take the work?

jetm commented:
@wsdjeg Sorry, I have been busy. I will take the job. Will submit a new change during the day.

Commit:
Add [SPC]bb map in fzf layer to nagivate in all buffers

fzf layer works better than denite and unite layers. This commit add
`[SPC]bb` map in fzf layer.

denite layer used to enable `[SPC]bb` to navigate in all buffers.
This change add same functionality but in fzf layer.

wsdjeg commented:
Thanks, merged. :)
